Documentation/gettinghelp.rst

```diff
@@ -4,18 +4,23 @@
 Getting Help
 ############
 
-We use GitHub issues to maintain a list of `Cilium Frequently Asked Questions (FAQ)`_.
-Check there to see if your question(s) is already addressed.
+Cilium is a project  with a growing community, so if you need help there a
+numerous ways to getting help:
 
-The best way to get help if you get stuck is to contact us on the
-`Cilium Slack channel <https://cilium.herokuapp.com>`_.
+**Cilium Frequently Asked Questions (FAQ)**: Cilium uses `GitHub
+tags <https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=label%3Akind%2Fquestion%20>`_
```

Comment:
Will this parse correctly despite the newline in the middle of the link?

Reply:
Working perfect ;-)